### PR TITLE
Add delete method for empty bucket

### PR DESCRIPTION
**Pull request by ShatalovYaroslav**

### CODE REVIEW

File: src/main/java/org/ow2/proactive/workflow_catalog/rest/controller/BucketController.java [hunk 1]
```diff
@@ -87,4 +89,11 @@ public PagedResources list(
         return bucketService.listBuckets(ownerName, pageable, assembler);
     }
 
+    @ApiOperation(value = "Delete an empty bucket", notes = "It's forbidden to delete the not empty bucket. You need to delete manually all workflows in the bucket before.")
```
Comment (mbenguig) [inline src/main/java/org/ow2/proactive/workflow_catalog/rest/controller/BucketController.java]: .. to delete a non-empty ...

**Review by laurianed — APPROVED** [review]

File: src/main/java/org/ow2/proactive/workflow_catalog/rest/controller/BucketController.java (hunk 1 above)
Comment (ShatalovYaroslav) [reply src/main/java/org/ow2/proactive/workflow_catalog/rest/controller/BucketController.java]: thanks.

File: src/main/java/org/ow2/proactive/workflow_catalog/rest/service/DeleteNotEmptyBucketException.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
+/*
+ * ProActive Parallel Suite(TM):
```
Comment (mbenguig) [inline src/main/java/org/ow2/proactive/workflow_catalog/rest/service/DeleteNotEmptyBucketException.java]: In the class name => DeleteNonEmptyBucketException.java

File: src/main/java/org/ow2/proactive/workflow_catalog/rest/service/DeleteNotEmptyBucketException.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+/**
+ * This Exception is thrown when a DELETE request for bucket has been
+ * received but such a Bucket is not empty (name + owner)
+ * The HTTP status is 403, 'Forbidden"
+ *
+ * @author ActiveEon Team
+ */
+@ResponseStatus(value = HttpStatus.FORBIDDEN)
+public class DeleteNotEmptyBucketException extends RuntimeException {
```
Comment (mbenguig) [inline src/main/java/org/ow2/proactive/workflow_catalog/rest/service/DeleteNotEmptyBucketException.java]: Idem  (and for the followings)..

**Review by mbenguig — APPROVED** [review]
Only if the other reviewer added some comments forcing you to update your commit, you could rename the NotEmpty.. occurencies.. but great job.

**Pull request merged**